scripts/cmake/CMakeSetup.cmake

```diff
@@ -1,14 +1,22 @@
+# Disallow in-source builds
```

Comment:
Why?

Reply:
It disallows running cmake in the source root directory only as this clutters your git project. You can of course still have `source/build` or similar. `build*/` is even git-ignored.